### PR TITLE
DTFS2-4536

### PR DESCRIPTION
**Pull request by eddieleeper**

### CODE REVIEW

File: portal-api/src/v1/gef/models/application.js [hunk 1]
```diff
@@ -19,17 +19,18 @@ class Application {
...
       this.updatedAt = Date.now();
-      this.comments = req.comments;
-      this.submissionType = req.submissionType;
-      this.submissionCount = req.submissionCount;
-      this.submissionDate = req.submissionDate;
-      this.ukefDealId = req.ukefDealId;
-      this.checkerId = req.checkerId;
+      // Only set properties if they are part of the request otherwise they get cleared
+      const updatable = ['comments', 'submissionType', 'submissionCount', 'submissionDate', 'ukefDealId', 'checkerId'];
+      Object.keys(req).forEach((key) => {
```
Comment (davehlewis) [inline portal-api/src/v1/gef/models/application.js:28]: Might be a bit easier to read to us `Object.entries` given key and value used?

**Review by davehlewis — APPROVED** [review]
Looks good, just a minor thing

File: portal-api/src/v1/gef/models/application.js (hunk 1 above)
Comment (eddieleeper) [reply portal-api/src/v1/gef/models/application.js:28]: Yeah - I thought about that - I'll try refactoring and see how it looks.

**Review by davehlewis — APPROVED** [review]
LGTM

**Pull request merged**